modules/backend/widgets/filter/partials/_scope_checkbox.htm

```diff
@@ -2,6 +2,6 @@
 <div
     class="filter-scope checkbox custom-checkbox"
     data-scope-name="<?= $scope->scopeName ?>">
-    <input type="checkbox" id="<?= $scope->getId() ?>" <?= $scope->value ? 'checked' : '' ?> />
+    <input type="checkbox" id="<?= $scope->getId() ?>" <?= $scope->value==1 ? 'checked' : '' ?> />
```

Comment:
Again, strict type comparison please. Also, why is this check necessary?

Reply:
@LukeTowers  it is not necessary.. done changing